en_US/deploy/cluster/create-cluster.md

```diff
@@ -1,140 +1,142 @@
+| `etcd`   | Create a cluster via etcd                 |
+| `k8s`    | Create a cluster via Kubernetes service   |
 
-Suppose there are two nodes, `emqx@s1.emqx.io` and `emqx@s2.emqx.io`, and we can follow the steps below to create a cluster for these two nodes manually:
+[^*]: The multicast discovery strategy has been deprecated and will be removed in future releases.
 
-1. Set the cluster discovery strategy to `manual`:
+## Before You Begin
 
-   ```bash
-   cluster {
-       ## Options: manual | static | mcast | dns | etcd | K8s
-       discovery_strategy  =  manual
-   }
-   ```
+Before creating an EMQX cluster, the following prerequisites should first be met:
 
-2. After the two nodes are started, run the join cluster command on one node:
+1. All nodes are set with a unique node name in the format of `name@host`, where host must be an IP address or fully qualified domain name (FQDN). For more information on configuring node names, see [Configure Node Names](#configure-node-names). 
+
+2. If there is a firewall or security group between nodes, ensure the cluster communication port has been opened. For details, see [Intra-cluster communication port](./security.md).
 
-   ```bash
-   $ ./bin/emqx_ctl cluster join emqx@s1.emqx.io
+3. For security concerns, you should change the default cookie settings to a secret cookie in `emqx.conf` on all nodes to join the cluster. Note: All nodes to join the cluster should use the same security cookie. For details about the magic cookie used, see [Distributed Erlang - Security](https://www.erlang.org/doc/reference_manual/distributed.html#security). 
 
-   Join the cluster successfully.
-   Cluster status: [{running_nodes,['emqx@s1.emqx.io','emqx@s2.emqx.io']}]
+   ```
+   node {
+     cookie = "<a secret cookie>"
+   }
    ```
 
-   :::tip
+:::
 
-   1. This command must be run on the node to join the cluster, that is, as a **request** rather than **invite**.
-   2. After `emqx@s2.emqx.io` joins `emqx@s1.emqx.io` to form a cluster, it will clear the local data and synchronize the data in `emqx@s1.emqx.io`.
-   3. When a node that already joined a cluster joins another cluster, it should first leave the current cluster.
-      :::
+Now you can begin your journey to create an EMQX cluster. 
 
-3. Query the cluster status on any node:
+## Configure Node Names
 
-   ```bash
-   $ ./bin/emqx_ctl cluster status
+Before creating a cluster, you need first to name the nodes to join the cluster. Suppose you want to create a cluster for 2 nodes deployed in `s1.emqx.io` and `s2.emqx.io` respectively, you can follow the steps below to create the cluster. 
 
-   Cluster status: [{running_nodes,['emqx@s1.emqx.io','emqx@s2.emqx.io']}]
-   ```
+Configure the node name in the `emqx.conf` configuration file of the 1st node, for example:
 
-4. You can let a node leave the cluster with leave or force-leave:
+```bash
+node.name = emqx@s1.emqx.io
+# or
+node.name = emqx@192.168.0.10
```

Comment:
Perhaps leave out static IPs from the examples, since we discourage their use.

Reply:
agree, so i will remove this line